draft-ietf-quic-tls.md

```diff
@@ -359,10 +360,12 @@ levels fit into the handshake process.
 
 ## Interface to TLS
 
-As shown in {{schematic}}, the interface from QUIC to TLS consists of three
+As shown in {{schematic}}, the interface from QUIC to TLS consists of four
 primary functions:
 
 - Sending and receiving handshake messages
+- Processing stored transport and application state from a 0-RTT capable session
```

Comment:
s/0-RTT capable session ticket/resumed session/

```suggestion
- Processing stored transport and application state from a resumed session
```

As before with the mention of tickets; and early data is already covered.